website/_docs/install/kylin_docker.md

```diff
@@ -3,8 +3,141 @@ layout: docs
+
+After the container is started, we can enter the container through the `docker exec` command. Of course, since we have mapped the specified port in the container to the local port, we can open the pages of each service directly in the native browser, such as: 
+
+- Kylin Web UI: [http://127.0.0.1:7070/kylin/login](http://127.0.0.1:7070/kylin/login)
+- Hdfs NameNode Web UI: [http://127.0.0.1:50070](http://127.0.0.1:50070/)
+- Yarn ResourceManager Web UI: [http://127.0.0.1:8088](http://127.0.0.1:8088/)
+- HBase Web UI: <http://127.0.0.1:60010>
+
+In the container, the relevant environment variables are as follows: 
+
+{% highlight Groff markup %}
+JAVA_HOME=/home/admin/jdk1.8.0_141
+HADOOP_HOME=/home/admin/hadoop-2.7.0
+KAFKA_HOME=/home/admin/kafka_2.11-1.1.1
+SPARK_HOME=/home/admin/spark-2.3.1-bin-hadoop2.6
+HBASE_HOME=/home/admin/hbase-1.1.2
+HIVE_HOME=/home/admin/apache-hive-1.2.1-bin
+KYLIN_HOME=/home/admin/apache-kylin-3.0.0-alpha2-bin-hbase1x
+{% endhighlight %}
+
+## Build image to verify source code modifications
+
+The docker image can also be used when developers have modified the source code and want to package, deploy, and verify the source code. First, we go to the docker directory under the root directory of the source and execute the script below to build the image and copy the source into the image.: 
+
+{% highlight Groff markup %}
+#!/usr/bin/env bash
+
+echo "start build kylin image base on current source code"
+
+rm -rf ./kylin
+mkdir -p ./kylin
+
+echo "start copy kylin source code"
+
+for file in `ls ../../kylin/`
+do
+    if [ docker != $file ]
+    then
+        cp -r ../../kylin/$file ./kylin/
+    fi
+done
+
+echo "finish copy kylin source code"
+
+docker build -t apache-kylin-standalone .⏎
+{% endhighlight %}
+
+Due to need to download and deploy various installation packages over the network, the entire build process can last for tens of minutes, depending on the network.
+
+When the image build is complete, execute the following command to start the container: 
+
+{% highlight Groff markup %}
+docker run -d \
+-m 8G \
+-p 7070:7070 \
+-p 8088:8088 \
+-p 50070:50070 \
+-p 8032:8032 \
+-p 8042:8042 \
+-p 60010:60010 \
+apache-kylin-standalone
+{% endhighlight %}
+
+When the container starts, execute the docker exec command to enter the container. The source code is stored in the container dir `/home/admin/kylin_sourcecode`, execute the following command to package the source code: 
+
+{% highlight Groff markup %}
+cd /home/admin/kylin_sourcecod
+build/script/package.sh
+{% endhighlight %}
+
+After the package is complete, an installation package ending in `.tar.gz` will be generated in the `/home/admin/kylin_sourcecode/dist` directory, such as `apache-kylin-3.0.0-alpha2-bin-hbase1x.tar.gz`. We can use this installation package to deploy and launch Kylin services such as:
```

Comment:
I am not sure, because I am not good at English. But I think *installation package* is strange in my mind, maybe binary package is better?